docs/modules/ROOT/pages/cluster-side-modules.adoc

```diff
@@ -26,7 +26,9 @@ Cluster-side modules include any code that must be deployed as Java code to the
 |Module|Description
 
 |link:https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/map/MapStore.html[MapStore]
-|Ingest data from an external data source into a Hazelcast map, or add data from a Hazelcast map to an external data source.
+|Ingest data from an external data source to your Hazelcast map, and add data from your Hazelcast map to an external data source.
```

Comment:
This should really say
```suggestion
|Load data from an external data source to your Hazelcast map when it's missing, and propagate updates to your Hazelcast map to an external data source.
```

Not sure about the working, but I think it conveys the right idea.

Reply:
Reworded